vin/app/controllers/vin/notes_controller.rb

```diff
@@ -1,77 +0,0 @@
-class Vin::NotesController < ApplicationController
-respond_to :json
-
-  # GET /vin/sub/:uid/shipments/:sid/notes
-  def index
-
-    @subscriber = Subscriber.find(params[:sub_id])
```

Comment:
Yeah, i already thought about that, but i was speeding up to have the whole skeleton of the use cases first, thanks for notting it so i wouldn't forget :)